pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -2203,8 +2203,8 @@ benchmarks! {
 		let mut candidate_count = 1u32;
 		let mut seed = USER_SEED;
 
-		// Create 100 collators
-		for i in 0..100 {
+		// Create collators up to MaxCandidates
+		for i in 0..(T::MaxCandidates::get() - 2) {
```

Comment:
why `- 2`?

Reply:
Because I need to create 2 extra collators a few steps below. I will update the comment, it might be a bit confusing as is right now.